### ISSUE TITLE
"latest" surveyor tag in docker and compatibility with latest version

### ISSUE THREAD

**Issue by cartmast** [nats-io/nats-surveyor#52]
The "latest" docker tag and the default [here](https://github.com/nats-io/nats-surveyor/blob/master/docker-compose/.env#L7) results in 

```
nats-surveyor | flag provided but not defined: -jetstream
nats-surveyor | Usage of /nats-surveyor:
nats-surveyor |   -a string
nats-surveyor |     	Network host to listen on. (default "0.0.0.0")
nats-surveyor |   -addr string
```

because of this [line](https://github.com/nats-io/nats-surveyor/blob/master/docker-compose/docker-compose.yml#L17).

My `.env` file:

```
NATS_SURVEYOR_SERVERS=nats://redacted-for-git-post.elb.us-west-2.amazonaws.com:4222
NATS_SURVEYOR_CREDS=/path/to/system/creds/sys.creds
NATS_SURVEYOR_SERVER_COUNT=10

PROMETHEUS_STORAGE=./storage/prometheus

SURVEYOR_DOCKER_TAG=latest
PROMETHEUS_DOCKER_TAG=latest
GRAFANA_DOCKER_TAG=latest
```

This can be avoided with using [5c1dedbc](https://hub.docker.com/r/synadia/nats-surveyor/tags?page=1&ordering=last_updated)... but this seems misleading based on default values in this code package and the README which does not list `-jetstream` as an option. Nor can you use the feature described [here](https://github.com/nats-io/nats-surveyor#jetstream) without the use of this flag...

So my question is: which tag should we be using? Can 5c1dedbc be trusted as stable? Or should we be using the "latest" in which there is not JetStream support?

**Comment by ripienaar** [comment]
Yes as is common the main of a repo is what is being developed vs the tags showing what is released.  As it stands we have not released a Surveyor with JS support.

**Comment by cartmast** [comment]
Okay. Thanks for the clarification.

With respect to JS, is all that is missing a dashboard per [issue #25](https://github.com/nats-io/nats-surveyor/issues/25)?

**Comment by ripienaar** [comment]
No we have quite a lot missing to make a full featured monitoring capability. We have added much to the server in terms of new data but do not yet gather that. 

If you want to play with what I have I can push a snapshot docker container with the commit ID of main. Leaving :latest as is

**Comment by cartmast** [comment]
If you don't mind that would be great. I appreciate your help here.
